### PR TITLE
feat: add retention advices

### PR DESCRIPTION
**Pull request by radireddy**
add vps and dedicated retention advices

ref: #MANAGER-7368

Signed-off-by: Ravindra Adireddy <ravindra.adireddy@ovhcloud.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | develop
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-7368
| License          | BSD 3-Clause

## Description

Added retention advices in VPS and dedicated server

### CODE REVIEW

**Review by frenautvh — APPROVED** [review]

File: packages/manager/modules/advices/src/advices/index.less [hunk 1]
```diff
@@ -16,11 +16,11 @@ ovh-advices {
   }
 
   oui-tile-definition.oui-tile__item {
-    border-top: 1px solid #ffffff;
+    border-top: 1px solid #aee5f3;
```
Comment (antleblanc) [inline packages/manager/modules/advices/src/advices/index.less:19]: Don't know if it has been already discuss, but might be interesting to move this into `ovh/ovh-ui-kit`.
What do you think about it?

**Review by antleblanc — APPROVED** [review]

File: packages/manager/modules/advices/src/advices/index.less (hunk 1 above)
Comment (radireddy) [reply packages/manager/modules/advices/src/advices/index.less:19]: Good suggestion. ovh-ui-kit has no utility for border. It has utility classes for background and text colour. Adding utility just for this does not justify the efforts. Shall we consider this once we have multiple use cases?

**Review by frenautvh — DISMISSED** [review]

**Review by varun257 — DISMISSED** [review]

**Review by ganeshkumar1989 — DISMISSED** [review]

**Comment by ovh-cds** [comment]
CDS Report translations-checker#19266.0 ✘  
* Stage 1
  * Check ✘

File: packages/manager/modules/advices/src/advices/index.less (hunk 1 above)
Comment (marie-j) [reply packages/manager/modules/advices/src/advices/index.less:19]: Bootstrap4 on the other hand provides border utilities.

**Review by varun257 — APPROVED** [review]

File: packages/manager/modules/advices/src/advices/index.less (hunk 1 above)
Comment (radireddy) [reply packages/manager/modules/advices/src/advices/index.less:19]: Yes it has border-primary, border-secondary etc. No class with color "aee5f3". Did you know any solution/workaround to change the color?

Comment (antleblanc) [reply packages/manager/modules/advices/src/advices/index.less:19]: Maybe guidelines around both tile/box components can/should evolve to handle such use case.

**Review by varun257 — DISMISSED** [review]

File: packages/manager/modules/advices/src/advices/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -1,3 +1,4 @@
 {
-  "advices_heading": "Nos conseils"
+  "advices_heading": "Nos conseils",
+  "advices_retention_heading": "Nos guides"
```
Comment (mohammed-zahaf) [inline packages/manager/modules/advices/src/advices/translations/Messages_fr_FR.json:3]: I know you don't touch that, but regarding others languages w have to add:
 ```suggestion
  "advices_retention_heading": "Nos guides :"
```

Comment (radireddy) [reply packages/manager/modules/advices/src/advices/translations/Messages_fr_FR.json:3]: The content is reviewed and approved. There is no semicolon at the end. I am not sure why translation team added it in other languages when its not there in source language.

Comment (mohammed-zahaf) [reply packages/manager/modules/advices/src/advices/translations/Messages_fr_FR.json:3]: Ok, so we should probably create another task to remove the extra semicolon for others languages.

**Review by mohammed-zahaf — APPROVED** [review]

**Review by antleblanc — DISMISSED** [review]

**Comment by antleblanc** [comment]
Rebasing on following branch `release-2021-w36`.

**Comment by antleblanc** [comment]
Rebased successfully on `release-2021-w36` branch.

**Review by antleblanc — APPROVED** [review]

**Pull request merged**